pkg/commands/commit_list_builder.go

```diff
@@ -288,8 +288,8 @@ func (c *CommitListBuilder) getLog(limit bool) string {
 		limitFlag = "-30"
 	}
 
-	c.Log.Warn(fmt.Sprintf("git log --oneline %s", limitFlag))
-	result, err := c.OSCommand.RunCommandWithOutput(fmt.Sprintf("git log --oneline %s", limitFlag))
+	c.Log.Warn(fmt.Sprintf("git log --oneline %s --abbrev=%d", limitFlag, 20))
```

Comment:
I'd remove this line because we will already log the command from within RunCommandWithOutput